internal/stanza/config.go

```diff
@@ -24,13 +26,19 @@ import (
 type BaseConfig struct {
 	configmodels.ReceiverSettings `mapstructure:",squash"`
 	Operators                     OperatorConfigs `mapstructure:"operators"`
+	Converter                     ConverterConfig `mapstructure:"converter"`
 }
 
 // OperatorConfigs is an alias that allows for unmarshaling outside of mapstructure
 // Stanza operators should will be migrated to mapstructure for greater compatibility
 // but this allows a temporary solution
 type OperatorConfigs []map[string]interface{}
 
+type ConverterConfig struct {
+	FlushTriggerCount uint          `mapstructure:"flush_trigger_count"`
+	FlushInterval     time.Duration `mapstructure:"flush_interval"`
```

Comment:
Sure, I'm ok with changing that 👍

Reply:
After taking a second look at it I'd go with `max_flush_count` and `flush_interval`.

While resetting the flush ticker (which isn't done as of now) makes the interval actually a "max" time duration between the flushes, it feels more natural to read the config with `flush_interval` in it rather than `max_flush_interval`.